File: backend/src/main/java/com/daedan/festabook/festival/dto/FestivalNotificationResponse.java
package com.daedan.festabook.festival.dto;

import com.daedan.festabook.festival.domain.FestivalNotification;

public record FestivalNotificationResponse(
        Long festivalNotificationId
) {

    public static FestivalNotificationResponse from(FestivalNotification festivalNotification) {
        return new FestivalNotificationResponse(
                festivalNotification.getId()
        );
    }
}

File: android/app/src/main/java/com/daedan/festabook/data/model/response/festival/FestivalNotificationResponse.kt
package com.daedan.festabook.data.model.response.festival

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class FestivalNotificationResponse(
    @SerialName("festivalNotificationId")
    val festivalNotificationId: Long,
)
